### PR TITLE
TeamView Patch Changelog

### PR DESCRIPTION
**Pull request by Kroronos**
- Team View Implemented
- SFrame SUserItemInspection Implemented
- Case Insensitive & Whitespace Delimited Sorting
- commentField SFrame implementation
- Miscellaneous resizing bugs fixed
- Show Items Assigned to User Click
- Listbox Select Event Bug fixed with listboxEventHandler

Issues Addressed In Path:
#47, #57, #64 (Partially)

### CODE REVIEW

**Comment by fallinbryan** [comment]
no easy way to approve on my phone. will be at laptop shortly

**Comment by Kroronos** [comment]
controller when passed as a parameter to an object should refer to the view one level above it.
Its included so as to bind tkinter objects to that frame (usually done in scrumblesFrames on the __init__ for the ScrumblesFrame Frame itself). 
The issue being I was starting to use it as a shortcut to get to master (controller.controller) and this wouldn't be a good thing if you nested a ScrumblesFrame.

File: source/developerHomeView.py [hunk 1]
```diff
@@ -42,52 +52,37 @@ def __init__(self, parent, controller, user):
...
-        self.assignedItems.clear()
-        for item in self.controller.dataBlock.items:
-            if item.itemUserID == userID:
-                self.assignedItems.append(item.itemTitle)
+    def assignedItemEvent(self, event):
+        for user in self.controller.dataBlock.users:
+            if user.userName == event.widget.get(tk.ANCHOR):
+                self.selectedUser = user
+                self.assignedItems = user.listOfAssignedItems
+                self.assignedItemList.importItemList(self.assignedItems)
```
Comment (fallinbryan) [inline source/developerHomeView.py]: Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Python3\lib\tkinter\__init__.py", line 1699, in __call__
    return self.func(*args)
  File "C:\Users\Bryan\Documents\GitHub\GroupProject\source\developerHomeView.py", line 46, in <lambda>
    source.bind('<<ListboxSelect>>', lambda event: self.eventHandler.handle(event))
  File "C:\Users\Bryan\Documents\GitHub\GroupProject\source\listboxEventHandler.py", line 26, in handle
    localEvent(event)
  File "C:\Users\Bryan\Documents\GitHub\GroupProject\source\developerHomeView.py", line 79, in listboxEvents
    self.assignedItemEvent(event)
  File "C:\Users\Bryan\Documents\GitHub\GroupProject\source\developerHomeView.py", line 60, in assignedItemEvent
    self.assignedItemList.importItemList(self.assignedItems)
AttributeError: 'list' object has no attribute 'importItemList'

Comment (fallinbryan) [reply source/developerHomeView.py]: To duplicate exception:
Go to Dev. Home View click around
Go to Backlog View
Right click an item and try to edit ,For some reason Item editor is broken.  will create new issue.
Go back to Dev Home View
Click an item

**Comment by fallinbryan** [comment]
can't duplicate bug now

**Pull request merged**